.github/workflows/fast.yml

```diff
@@ -6,6 +6,10 @@ on: workflow_call
 permissions:
   contents: read
 
+env:
+  BUILD_DIR : "${{github.workspace}}/build"
```

Comment:
we could use BUILD_DIR in VCPKG paths

Reply:
We cannot, it would be a syntax error, because GHA does not allow to use env vars inside a definition of another env var.

Reply:
ok I see, thanks